build.gradle.kts

```diff
@@ -40,7 +40,7 @@ liquibase {
         register("main") {
             arguments = mapOf(
                     "changeLogFile" to "db/changelog/db.changelog-master.xml",
-                    "url" to "jdbc:mysql://$databaseUrl/save_cloud",
```

Comment:
Also let's move `secretes` file to home dir

Reply:
Fixed